src/bin/tikv-ctl.rs

```diff
@@ -411,6 +411,12 @@ trait DebugExecutor {
     /// Recover the cluster when given `store_ids` are failed.
     fn remove_fail_stores(&self, store_ids: Vec<u64>);
 
+    /// Remove region.
+    fn remove_region(&self, region_id: u64);
+
+    /// Init a new empty region on the store with given infos.
+    fn init_empty_region(&self, Arc<SecurityManager>, &PdConfig, u64);
```

Comment:
It's not clear what the parameters mean here. Could we name them like in the functions above?